recipes-samples/packagegroups/packagegroup-rpb-tests.bb

```diff
@@ -12,6 +12,7 @@ PACKAGES = "\
 # contains basic dependencies, that don't need graphics/display
 RDEPENDS_packagegroup-rpb-tests = "\
     packagegroup-core-buildessential \
+    packagegroup-core-tools-debug \
```

Comment:
tools-debug is 'expected' to be an IMAGE_FEATURE. the fact that tools-debug maps to packagegroup-core-tools-debug is an 'implementation' detail, we should problably not rely on it.

FEATURE_PACKAGES_tools-debug = "packagegroup-core-tools-debug"